cmd/api/src/analysis/ad/adcs_integration_test.go

```diff
@@ -100,8 +99,57 @@ func TestADCSESC1(t *testing.T) {
 				require.True(t, results.Contains(harness.ADCSESC1Harness.Group47))
 
 			}
+
+			// Domains 1 and 4 have multiple ADCSESC1 edges so we will just test edge comp of Domains 2 and 3
+
+			// Domain 2 Edge Composition
+			if edge, err := tx.Relationships().Filterf(func() graph.Criteria {
+				return query.And(
+					query.Kind(query.Relationship(), ad.ADCSESC1),
+					query.Equals(query.EndID(), harness.ADCSESC1Harness.Domain2.ID),
+				)
+			}).First(); err != nil {
+				t.Fatalf("error fetching esc1 domain 2 edges in integration test; %v", err)
+			} else {
+				comp, err := ad2.GetADCSESC1EdgeComposition(context.Background(), db, edge)
+				assert.Nil(t, err)
+
+				esc12nodes := comp.AllNodes()
+				assert.Len(t, esc12nodes, 7)
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.Group22))
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.CertTemplate2))
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.EnterpriseCA21))
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.EnterpriseCA23))
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.AuthStore2))
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.RootCA2))
+				require.True(t, esc12nodes.Contains(harness.ADCSESC1Harness.Domain2))
+			}
+
+			// Domain 3 Edge Composition
+			if edge, err := tx.Relationships().Filterf(func() graph.Criteria {
+				return query.And(
+					query.Kind(query.Relationship(), ad.ADCSESC1),
+					query.Equals(query.EndID(), harness.ADCSESC1Harness.Domain3.ID),
+				)
+			}).First(); err != nil {
+				t.Fatalf("error fetching esc1 edges in integration test; %v", err)
+			} else {
+				comp, err := ad2.GetADCSESC1EdgeComposition(context.Background(), db, edge)
+				assert.Nil(t, err)
+
+				esc13nodes := comp.AllNodes()
+				assert.Len(t, esc13nodes, 6)
+				require.True(t, esc13nodes.Contains(harness.ADCSESC1Harness.Group32))
```

Comment:
nit: I thought this was related to escalation 13 from the var name. Maybe we can name these `domain3Nodes` or something instead?

Reply:
Ha, fun coincidence, will update!